12-make_inverse.py

```diff
@@ -38,9 +39,9 @@ def run_inverse(subject, session=None):
                                        space=config.space)
 
     fname_ave = op.join(deriv_path, bids_basename + '-ave.fif')
-    fname_fwd = op.join(deriv_path, bids_basename + '-fwd.fif')
-    fname_cov = op.join(deriv_path, bids_basename + '-cov.fif')
-    fname_inv = op.join(deriv_path, bids_basename + '-inv.fif')
+    fname_fwd = op.join(deriv_path, f'{bids_basename}_{kind}-fwd.fif')
+    fname_cov = op.join(deriv_path, f'{bids_basename}_{kind}-cov.fif')
```

Comment:
why also for the cov?

Reply:
To handle the case where the user asked us to compute the MEG cov based on an empty-room recording. The EEG cov will still be computed from the measurement data.
(Currently working on this in another branch)

Or do you have better idea?

Reply:
you must use only one noise cov for source imaging. As long as EEG is present you must use prestim. let's talk it will be easier.